### PR TITLE
Add section on clock priority levels in SE (#29)

### LINKED ISSUE #29: Move non-normative text from 5.1.1 to IG
`_[A clock's <<priority>> information is required to support the scheduled execution of several model partitions._
_In particular the <<preemption-support, preemption support>> is based on these priorities._
_It is therefore important to restrict the number of distinct priority levels for an FMU to available priority levels on the target platform and/or to avoid unnecessary computational overhead._
_A common number of different priority levels is, e.g., 100 (0 to 99), as defined in Linux based operating systems.]_`

Note: already removed in standard text.

### PR DESCRIPTION
**Pull request by pmai**
Closes #29.

### CODE REVIEW

**Comment by pmai** [comment]
Note that I have copied over the remaining non-normative sections in 5.1.1, too, since they seem related in order to understand the full role of priorities. @MBlesken please check if it would make sense to also remove them in the standard, or to leave this information duplicated, which would be fine be me.

**Review by andreas-junghanns — APPROVED** [review]

File: fmi-guide/8___scheduled-execution.adoc [hunk 1]
```diff
@@ -5,3 +5,18 @@
 The scheduled execution interface type is intended for use cases, where the ability to directly control the scheduling of internal time partitions of multiple FMUs from the outside is considered necessary.
 
 Note that in cases where the actual direct control of the runtime of internal time partitions is not required, the use of clocked co-simulation or model exchange interface types may be more appropriate.
+
+=== Priority Levels of Clocks
+
+In scheduled execution clocks can be assigned priority information through the `priority` attribute on the clock variable.
```
Comment (MBlesken) [inline fmi-guide/8___scheduled-execution.adoc:11]: *must* be assigned

File: fmi-guide/8___scheduled-execution.adoc [hunk 2]
```diff
@@ -5,3 +5,18 @@
...
+
+A clock's priority information is required to support the scheduled execution of several model partitions, especially in a preemptive scheduling regime.
```
Comment (MBlesken) [inline fmi-guide/8___scheduled-execution.adoc:13]: "*especially* in a preemptive scheduling regime." is confusing: SE doesn't work without priorities. Preemption can be regarded as just another feature.

**Review by MBlesken — CHANGES_REQUESTED** [review]
Minor remarks

**Comment by MBlesken** [comment]
> Note that I have copied over the remaining non-normative sections in 5.1.1, too, since they seem related in order to understand the full role of priorities. @MBlesken please check if it would make sense to also remove them in the standard, or to leave this information duplicated, which would be fine be me.


@pmai Can we look at it together in a design meeting? I'm not sure where to get orientation from to decide on the level that makes sense to leave and to move....

**Pull request merged**